ogr/abstract.py

```diff
@@ -866,6 +874,16 @@ def can_merge_pr(self, username: str) -> bool:
         """
         raise NotImplementedError()
 
+    def add_user(self, user: str, access_level: AccessLevel) -> None:
+        """
+        Add User
+
+        :param user: str
```

Comment:
Can you please tweak the docstring a bit? Just a sentence and the `user` parameter description. (e.g What is expected to be in `user`? `id`, `username`, `email`...?)